docs/documentation/server_admin/topics/admin-console-permissions/fine-grain-v2.adoc

```diff
@@ -0,0 +1,227 @@
+those are used for evaluation.
+
+When checking whether an admin can view a specific user, all policy types are taken into account.
+
+==== Evaluation conflict resolution
+
+Permissions can have multiple policies associated with them. As the authorization model evolves, it is common for some policies within a permissionor 
+even different permissions related to a specific resource to conflict.
+
+The evaluation outcome will be "denied" whenever any permission is evaluated to "DENY." If there are multiple permissions related to the same resource, 
+all of them must grant access in order for the outcome to be "granted."
+
+IMPORTANT: Fine-grained admin permissions allow you to set up permissions for individual resources or for the resource type itself (e.g., all users, 
+all groups, etc.). If a permission or permissions related to a specific resource exist, the "all-resource" permission is *NOT* taken into account 
+during evaluation. If no specific permission exists, the fallback is to the "all-resource" permission. This approach helps solve scenarios like 
+allowing members of the `realm-admins` group to manage members of realm groups, but preventing them from managing members of the `realm-admins` group 
+themselves.
+
+==== Common use cases
+
+TODO: add more common use cases
+
+Let's say an admin wants to allow a group of admins to `manage` all users in the realm except those, which belongs to the
+administrators group. In our example, we have a realm called `test` and a group called `test-admins`. 
+
+===== Permission setup
+
+* Navigate to the `Permissions` tab in the Admin Console.
+* Click `Create permission` button and choose `Users` resource type
+* Fill the name, e.g. `"Allow managing all realm users"`
+* Choose `manage` authorization scope, keep checked `All Users`
+* Create a condition, which needs to be met to get an access by clicking `Create new policy`
+* Fill name `"Allow test-admins"`, select `Group` as `Policy type`. 
+* Click `Add groups` button and select `test-admins` group, click `Save`
+* Click `Save` on `"Create permission"` page
+
+The created permission, allows to manage all users in the realm for members of the `test-admins` group. 
+Let's exlude the members of the group itself, so that `test-admins` cannot manage other admins.
+
+* Create new permission by clicking `Create permission` button
+* This time choose `Groups` resource type
+* Fill the name, e.g. `"Disallow managing test-admins"`
+* Choose `manage-members` authorization scope.
+* Select `Specific Groups` and choose `test-admins` group
+* `Create new policy` of type `Group`
+* Fill the name `"Disallow test-admins"` and select `test-admins` group
+* Switch to `Negative Logic` for the policy, `Save` the policy
+* `Save` the permission
+
+==== Full list of permissions
+
+TODO this section needs review.
+
+This chapter defines the whole list of permission types that can be described for a realm.
+
+===== Client
+
+view::
```

Comment:
IMO, a table is better to list all the scopes/permissions that can be set to a resource.